lib/reporters/verbose.js

```diff
@@ -137,15 +140,17 @@ class VerboseReporter {
 			case 'test-failed':
 				this.failures.push(evt);
 				this.writeTestSummary(evt);
+				this.removeTestRunning(evt);
 				break;
 			case 'test-passed':
 				if (evt.knownFailing) {
 					this.knownFailures.push(evt);
 				}
 				this.writeTestSummary(evt);
+				this.removeTestRunning(evt);
 				break;
 			case 'timeout':
-				this.lineWriter.writeLine(colors.error(`${figures.cross} Exited because no new tests completed within the last ${evt.period}ms of inactivity`));
+				this.lineWriter.writeLine(colors.error(`${figures.cross} Exited because no new tests completed within the last ${evt.period}ms of inactivity\n${this.runningTestSummary()}`));
```

Comment:
`timeout` applies to the set of test files that are currently running. However unless `--fail-fast` is set, AVA will continue to run remaining test files. This means multiple timeouts may occur.

We should include the timed out files when the event is emitted: https://github.com/avajs/ava/blob/55e40213398dbcfd7f9941697e556b335f5229f2/api.js#L77

Then if you pass the `evt` to `runningTestSummary()` we could print the files appropriate for each particular timeout.